IT/src/test/java/com/microsoft/gctoolkit/integration/aggregation/CollectionCycleCountsAggregation.java

```diff
@@ -0,0 +1,12 @@
+package com.microsoft.gctoolkit.integration.aggregation;
+
+import com.microsoft.gctoolkit.aggregator.Aggregation;
+import com.microsoft.gctoolkit.aggregator.Collates;
+import com.microsoft.gctoolkit.event.GarbageCollectionTypes;
+
+@Collates(CollectionCycleCountsAggregator.class)
+public interface CollectionCycleCountsAggregation extends Aggregation {
```

Comment:
It is unfortunate that these aggregators/aggregations had to be duplicated here. Is it really worth having the duplicates just to have TestMain, especially when TestMain doesn't get run by default as a unit test. 

I think TestMain should be moved back to sample and these duplications removed.

Reply:
I held my nose when copying these classes into the IT module. This actually turns out to be a useful integration test. Having TestMain call Main wasn't going to work. Happy to explore other options